packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -389,4 +389,57 @@ describe('RichTextViewer', () => {
             ).toEqual(['John Doe', 'Mary Wilson']);
         });
     });
+
+    describe('getMentionedHref() for viewer mentions', () => {
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            pageObject = new RichTextViewerPageObject(element);
+            await connect();
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
```

Comment:
I just noticed that every `describe` block in this file has its own identical `beforeEach` and `afterEach`. Just add the call to `connect()` in the top-level `beforeEach`, then add the `afterEach` with `disconnect()` at the top level. Then you can delete all these duplicates in the `describe` blocks. As it is now, `setup()` is getting called and the page object is being created two times per test.

Reply:
Removed `beforeEach` and `afterEach` other than top level places.